tests/CSharp/CSharp.Tests.cs

```diff
@@ -1485,7 +1486,14 @@ public void TestAnonymousMemberNameCollision()
     [Test]
     public void TestStringMarshall()
     {
-        Assert.IsTrue(StringMarshall.CSharpString8(StringMarshall.CSharpString));
-        Assert.IsTrue(StringMarshall.CSharpString16(StringMarshall.CSharpString));
+        const string cs = "C#";
```

Comment:
Could you please test with a longer string which also contains non-ASCII characters? You can use https://github.com/mono/CppSharp/blob/master/tests/Common/Common.Tests.cs#L974 as a reference.